### PR TITLE
add intersections, labels for figures

### PR DESCRIPTION
**Pull request by Fenik2305**

### CODE REVIEW

File: package/qt_widgets/plotter_widget.py [hunk 1]
```diff
@@ -70,17 +153,14 @@ def view_zx(self):
...
-    def zoom_out(self):
-        self.zoom -= 1
-        self.plotter.camera.Zoom(self.zoom)
+    def blur(self):
+        self.plotter.add_blurring()
+    
+    def remove_blur(self):
+        self.plotter.remove_blurring()
 
     def take_screenshot(self, file_name='untitled.png'):
```
Comment (H3llAbyss) [inline package/qt_widgets/plotter_widget.py:162]: No file name choice? What if untitled.png already exists is photos\?

File: package/qt_widgets/plotter_widget.py [hunk 2]
```diff
@@ -1,56 +1,139 @@
...
+        self.update_camera()
 
     def update_camera(self):
-        x_min, x_max, y_min, y_max, z_min, z_max = self.plotter.bounds
-        new_position = tuple([(x_min + x_max) / 2, (y_min + y_max) / 2, (z_min + z_max) / 2])
-        self.plotter.fly_to(new_position)
+        self.plotter.view_isometric()
+
+    def get_bounds(self):
+        return self.plotter.bounds
```
Comment (H3llAbyss) [inline package/qt_widgets/plotter_widget.py:136]: Which data exactly will be returned? Please add -> to the function

**Review by H3llAbyss — CHANGES_REQUESTED** [review]
Please consider updating take_screenshot function. Also, some screenshots of funtionality would be really great

File: package/qt_widgets/plotter_widget.py [hunk 3]
```diff
@@ -1,56 +1,139 @@
...
         self.plotter.window_size = window_size
+        
+        self.plotter.enable_depth_peeling()
 
         self.plotter.add_axes()
         self.plotter.show_grid()
-        self.plotter.camera.Zoom(self.zoom)
-        self.plotter.enable_terrain_style()
+        self.plotter.enable_terrain_style(mouse_wheel_zooms=True)
+        self.plotter.view_isometric()
```
[the remark this answers is not in the document]
Comment (Fenik2305) [reply package/qt_widgets/plotter_widget.py:18]: view isometric returns the camera to the initial position, its use during initialization is used to correct the plotter display at the initial stage

**Pull request merged**